returnn/search.py

```diff
@@ -56,6 +57,9 @@ def __init__(
 
         self.model_checkpoint = model_checkpoint
 
+        assert output_mode in ["py", "txt"]
+        self.output_mode = output_mode
```

Comment:
maye we could pass this directly to the `create_returnn_config`!? so that if necessary the `create_returnn_config` could also easily be called from outside the job?

Reply:
Yes, this would be nice, but we can also do this in another PR, that one would not break anything.